framework/scripts/find_hung_process.py

```diff
@@ -331,6 +331,7 @@ def __init__(self):
         self.host_and_frames_re = re.compile('^(Host.*?)\n(.*)', re.M | re.S)
         self.memory_re = re.compile(r'0x[0-9a-f]*')    # Pattern to match (and remove) memory addresses
         self.frame_num_re = re.compile(r'^#\d.', re.M) # frame numbers IMPORTANT: Always match 2 characters due to fixed width formatting!
+        self.param_val_re = re.compile(r'(\w+?)=\S+(?=(?:,|\)))') # Pattern to match (and ignore) specific param values
```

Comment:
```suggestion
        self.param_val_re = re.compile(r'(\w+?)=\S+(?=(?:,|\)))') # Pattern to match (and ignore the others) specific param values
```

Reply:
No, this is not what I want to say. If anything dropping the parentheses would be the only thing that makes sense. I just want to ignore parameter values.

Reply:
alright then
```
Pattern to match and ignore parameter values for a specific parameter of any routine
```

Reply:
Not a specific parameter value, we either pay attention to all of them, or none of them.

Reply:
then dont say specific ?
```
Pattern to match and ignore parameter values for all parameters of any routine
```